src/main/java/org/demo/controller/CxboxRestController.java

```diff
@@ -63,6 +64,7 @@ public enum CxboxRestController implements EnumBcIdentifier {
 		contactPickListPopup(meetingEdit, ContactPickListService.class),
 
 	sale(SaleReadService.class),
+	saleClientList(clientEdit, SaleReadService.class),
```

Comment:
move it under parent (clientEdit bc) and use indentatation to show it is child bc